third_party/terraform/resources/resource_compute_backend_service.go.erb

```diff
@@ -92,6 +92,11 @@ func resourceComputeBackendService() *schema.Resource {
 								},
 							},
 						},
+						"signed_url_cache_max_age_sec": {
```

Comment:
drive-by: This file is getting generated by https://github.com/GoogleCloudPlatform/magic-modules/pull/1595, so adding this will be a matter of removing an exclude if you don't mind holding this bit.

Reply:
1595 is in, so we need to do this through MM now